### PR TITLE
refactor a qt dims test to not use the viewer

### PR DESCRIPTION
**Pull request by sofroniewn**
# Description
This short PR just moves one of our tests using `make_napari_viewer` out of the individual `qt_dims` tests and up to the `qt_viewer` tests, as a good rule of thumb, I think a test using `make_napari_viewer` shouldn't be at the level of the individual widget even if it is testing something focused on that widget as it is importing the whole viewer and so could fail for larger reasons.

## Type of change

- [x] Refactor

### CODE REVIEW

**Review by tlambert03 — APPROVED** [review]

**Review by tlambert03 — COMMENTED** [review]
fine with me.  In the future, I think rather than moving these tests up higher, we should aim to not use `make_test_viewer` at all.  But rather test things in isolation.   here for instance, we're mostly testing `viewer.dims`, which _might_ be doable without the viewer at all

**Comment by sofroniewn** [comment]
> here for instance, we're mostly testing viewer.dims, which might be doable without the viewer at all

yeah, that's a good idea - i can give that a go before merge, we might be able to easily test in isolation

**Comment by sofroniewn** [comment]
Ok @tlambert03 I have updated this PR based on your input and dropped `make_napari_viewer` from the test. I will merge this tomorrow unless hearing anything further. Thanks!!

**Review by tlambert03 — APPROVED** [review]

**Pull request merged**
